testing/pcl/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=pcl
+pkgver=1.8.1
+pkgrel=0
+pkgdesc="Point Cloud Library (PCL)"
+url="https://github.com/PointCloudLibrary/pcl"
+arch="all"
+license="BSD"
+options="!check"
+depends=""
+makedepends="cmake eigen-dev boost-dev flann-dev"
```

Comment:
It seems this package needs `flann-dev` which is not yet in aports.

Reply:
To make travis happy you can combine commits into a single PR.

Reply:
@clandmeter only problem is that the pcl build takes quite a while and will timeout if combined. This is how I originally had it structured.